recipe/meta.yaml

```
@@ -12,32 +12,37 @@ source:
   sha256: {{ sha256 }}
 
 build:
+  skip: true  # [py<38]
   number: 0
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
   host:
     - python
     - pip
+    - setuptools
+    - wheel
   run:
     - python
     - enum34  # [py2k]
     - pycodestyle
     - flake8
 
 test:
+  requires:
+    - pip
   commands:
+    - pip check
     - flake8 --version
   imports:
     - flake8_import_order
 
 about:
   home: https://github.com/PyCQA/flake8-import-order
-  license: LGPL-3.0
+  license: LGPL-3.0-only
```

```suggestion
  license: LGPL-3.0-or-later
```
Looks like it should be or later:
https://github.com/PyCQA/flake8-import-order/blob/master/COPYING#L153